ci/konflux_minikube_e2e_tests.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -exv
+
+mkdir -p /container_workspace/bin
+
+export KUBEBUILDER_ASSETS=/container_workspace/testbin/bin
+
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
How is this working when hermetic builds are set to true? We should clarify with konflux team because, either we are not enforcing hermetic builds and we think we are, or there is a glaring bug/issue with the hermetic build system. Either way the future of this needs to have kubectl as a trusted artifact.